Agreements/External/S_RR.txt

```diff
@@ -0,0 +1,3 @@
+FIR_COPX:*:*:LELNA:LFRD:*:London AC Worthing:LFRR Low:*:25000:LELNA
+
+FIR_COPX:*:*:LELNA:LFRC:*:London AC Worthing:LFRR Low:*:12000:LELNA
```

Comment:
Probably not an issue but couldn't find this in the LoA.

Reply:
Yeah I found it in a reference doc so added in here in case it was going to be added, but not sure it will be. Will remove now.